private/model/api/behavior.go

```diff
@@ -0,0 +1,320 @@
+			val += fmt.Sprintf("\nawstesting.Assert")
+
+			switch assertionName {
+			case "requestMethodEquals":
+				val += fmt.Sprintf("RequestMethodEquals(t, %q, req.HTTPRequest.Method)", assertionContext)
+			case "requestUrlMatches":
+				val += fmt.Sprintf("RequestURLMatches(t, %q, req.HTTPRequest.URL.String())", assertionContext)
+			case "requestUrlPathMatches":
+				val += fmt.Sprintf("RequestURLPathMatches(t, %q, req.HTTPRequest.URL.EscapedPath())", assertionContext)
+			case "requestUrlQueryMatches":
+				val += fmt.Sprintf("RequestURLQueryMatches(t, %q, req)", assertionContext)
+			case "requestHeadersMatch":
+				val += fmt.Sprintf("RequestHeadersMatch(t, %#v, req)", assertionContext)
+			case "requestBodyEqualsBytes":
+				val += fmt.Sprintf("RequestBodyEqualsBytes(t, %q, req)", assertionContext)
+			case "requestBodyEqualsJson":
+				val += fmt.Sprintf("RequestBodyEqualsJSON(t, %#v, req)", assertionContext)
+			case "requestBodyMatchesXml":
+				val += fmt.Sprintf("RequestBodyMatchesXML(t, %q, req, %v)", assertionContext, c.Request.EmptyShapeBuilder(&op.InputRef))
+			case "requestBodyEqualsString":
+				val += fmt.Sprintf("RequestBodyEqualsString(t, %q, req)", assertionContext)
+			case "requestIdEquals":
+				val += fmt.Sprintf("RequestIDEquals(t, %q, req.RequestID)", assertionContext)
+			case "responseDataEquals":
+				val += fmt.Sprintf("ResponseDataEquals(t, %v, resp)", c.BuildOutputShape(&op.OutputRef))
+			case "responseErrorDataEquals":
+				val += fmt.Sprintf("ResponseErrorDataEquals(t, %#v, err)", assertionContext)
+			case "responseErrorIsKindOf":
+				val += fmt.Sprintf("ResponseErrorIsKindOf(t, %q, err)", assertionContext)
+			case "responseErrorMessageEquals":
+				val += fmt.Sprintf("ResponseErrorMessageEquals(t, %q, err)", assertionContext)
+			case "responseErrorRequestIdEquals":
+				val += fmt.Sprintf("ResponseErrorRequestIDEquals(t, %q, err)", assertionContext)
+			default:
+				panic("Invalid assertion key")
+			}
+		}
+	}
+	return val
+}
+
+// Returns a value to set Credentials
+func (c Case) ConfigurationString(T Tests) string {
+	region := T.Defaults.Env["AWS_REGION"]
+	accessKeyId := T.Defaults.Env["AWS_ACCESS_KEY"]
+	secretAccessKey := T.Defaults.Env["AWS_SECRET_ACCESS_KEY"]
+	endpointUrl := ""
+	if len(c.LocalConfig) > 0 {
+		for key, val := range c.LocalConfig {
+			switch key {
+			case "region":
+				region = val
+			case "accessKeyId":
+				accessKeyId = val
+			case "secretAccessKey":
+				secretAccessKey = val
+			case "endpointUrl":
+				endpointUrl = val
+			}
+		}
+	} else {
+		for key, val := range c.LocalEnv {
+			switch key {
+			case "AWS_REGION":
+				region = val
+			case "AWS_ACCESS_KEY":
+				accessKeyId = val
+			case "AWS_SECRET_ACCESS_KEY":
+				secretAccessKey = val
+			case "EndPointURL":
+				endpointUrl = val
+			}
+		}
+	}
+	return fmt.Sprintf("\n\t\tRegion: aws.String(%#v),\n\t\tCredentials: credentials.NewStaticCredentials(%#v, %#v, %#v),",
```

Comment:
The tabs (`\t` ) can probably be left otu of this because `gofmt` will add these to the code.  Alternatively you could use a multi ine string to represent this that might be easier to read and update.

```go
const cfgFmt = `
Region: aws.String(%#v),
Credentials: credentials.NewStaticCredentials(%#v, %#v, %#v),
`

return fmt.Sprintf(cfgFmt, ...)
```